package.json

```diff
@@ -53,7 +53,9 @@
     "typescript": "~3.6.0"
   },
   "dependencies": {
+    "@types/fs-extra": "^8.0.1",
```

Comment:
`@types` should go to `devDependencies`. Also, `fs-extra`, if it's only used in tests, should also go to `devDependencies`.

Reply:
done.